OpenOversight/app/models/database.py

```diff
@@ -63,22 +62,48 @@
 )
 
 
-class Department(BaseModel):
-    __tablename__ = "departments"
-    id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(255), index=False, unique=False, nullable=False)
-    short_name = db.Column(db.String(100), unique=False, nullable=False)
-    state = db.Column(db.String(2), server_default="", nullable=False)
+@declarative_mixin
+class TrackUpdates:
+    """Add columns to track the date of and user who created and last modified
+    the object.
+    """
+
     created_at = db.Column(
         db.DateTime(timezone=True),
         nullable=False,
         server_default=sql_func.now(),
         unique=False,
     )
-    created_by = db.Column(
-        db.Integer, db.ForeignKey("users.id", ondelete="SET NULL"), unique=False
+    last_updated_at = db.Column(
+        db.DateTime(timezone=True),
+        nullable=True,
+        unique=False,
```

Comment:
Since we are always adding a value to `last_updated_at` when we create it, we could add make it have the same setup as `created_at` (non-nullable, defaults to `sql_func.now()`).

Reply:
Updated!